_shows/16_17/revolt_she_said.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Revolt. She Said. Revolt Again."
+playwright: Alice Birch
+student_written: false
+season: In House
+season_sort: 140
+period: Spring
+venue: New Theatre
+date_start: 2017-03-01
+date_end: 2017-03-04
+
+cast:
+- role: Dinah
+  name: Daniella Finch
+- role: Grandma
+  name: Lois Baglin
+- role: Agnes
+  name: Chloe Richardson
+- role: Multi-Role
+  name: Harry Pavlou
```

Comment:
Shouldn't all the cast have a  'multi-role' after them, Dinah, Grandma and Agnes are only their characters in Act 3